README.md

```diff
@@ -114,69 +90,38 @@ ie: [`5.0.0`](https://github.com/SonarSource/gh-action_release/releases/tag/5.0.
 
 ### Branches
 
-The `master` branch shall not be referenced by end-users.
```

Comment:
I prefer to be more directive like before:
```
The `master` branch shall not be referenced by end-users.
```

Like that RE can reference it for valid scenarios, and end users not, it is very explicit.

The new variant, do not define any target audience allowed to do it:
"you should not open pandora's box, but you can if you take care." 

It's less clear to me, up to you